packages/medusa/src/api-v2/admin/draft-orders/route.ts

```diff
@@ -0,0 +1,76 @@
+import { createOrdersWorkflow } from "@medusajs/core-flows"
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  ContainerRegistrationKeys,
+  OrderStatus,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaRequest,
+  MedusaResponse,
+} from "../../../types/routing"
+import { defaultAdminOrderFields } from "./query-config"
+import { AdminPostDraftOrdersReq } from "./validators"
+
+export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
+  const remoteQuery = req.scope.resolve(ContainerRegistrationKeys.REMOTE_QUERY)
+
+  const queryObject = remoteQueryObjectFromString({
+    entryPoint: "order",
+    variables: {
+      filters: {
+        ...req.filterableFields,
+        status: OrderStatus.DRAFT,
+      },
+      order: req.listConfig.order,
+      skip: req.listConfig.skip,
+      take: req.listConfig.take,
+    },
+    fields: defaultAdminOrderFields,
+  })
+
+  const { rows: orders, metadata } = await remoteQuery(queryObject)
+
+  res.json({
+    orders,
```

Comment:
thought: Would it make sense to still return these as `draft_orders`? If not, this is breaking, so should be documented.